src/components/main/AppCard.tsx

```diff
@@ -39,77 +41,58 @@ const AppCard = ({
   onClick,
   selected = false,
 }: Props): JSX.Element => (
-  <Card
-    sx={{
-      width: '100%',
-      outline: selected ? '2px solid #5050d2' : '',
-    }}
-    onClick={onClick}
-    id={buildItemFormAppOptionId(id ?? name)}
-  >
-    <StyledCardActionArea>
-      <Grid2
-        container
-        direction="row"
-        alignItems="center"
-        width="100%"
-        spacing={[0, 1]}
-      >
-        <Grid2 xs={2} p={[0, 1.5]}>
-          <Box
-            component="img"
-            src={image ?? defaultImage}
-            alt={name}
-            width="100%"
-            display="flex"
-            flexShrink={0}
-            minWidth={0}
-            borderRadius={1}
-            overflow="hidden"
-          />
-        </Grid2>
-        <Grid2 xs={10}>
-          <CardContent
-            sx={{
-              pt: 1,
-              paddingBottom: '0 !important',
-              minWidth: 0,
-            }}
-          >
-            <Typography
-              fontWeight="bold"
-              gutterBottom
-              variant="body1"
-              component="div"
-              width="100%"
-              noWrap
-            >
-              {name ?? <Skeleton />}
-            </Typography>
-            <Typography
-              variant="body2"
-              color="text.secondary"
-              width="100%"
+  <Tooltip title={description || ''}>
+    <Card
+      sx={{
+        width: '100%',
+        maxHeight: '80px',
+        flex: '1 1 calc(50% - 8px)',
+        outline: selected ? '2px solid #5050d2' : '',
+      }}
+      onClick={onClick}
+      id={buildItemFormAppOptionId(id ?? name)}
+    >
+      <StyledCardActionArea>
+        <Grid container direction="row" alignItems="center">
+          <Grid xs={1.5} display="flex" justifyContent="center">
+            <Box
+              component="img"
+              src={image ?? defaultImage}
+              alt={name}
+              width="80%"
+              flexShrink={0}
+              minWidth={0}
+              borderRadius={1}
+              overflow="hidden"
+              justifyContent="center"
+            />
+          </Grid>
+          <Grid xs={10.5}>
+            <CardContent
               sx={{
-                display: { xs: 'none', sm: 'block', md: 'block' },
+                pt: 1,
+                paddingBottom: '0 !important',
```

Comment:
Was the `important` necessary?

Reply:
I am afraid so, the Card Content component has a built-in weird bottom padding that is lager than the other dimensions. The `important` was used to override this behavior.